1.8/usage/storage/persistent-volume.md

```diff
@@ -75,6 +75,14 @@ The second volume is a persistent volume with a `containerPath` that matches the
 
 For a complete example, see the [Running stateful MySQL on Marathon](#stateful-sql) section.
 
+## Creating a stateful application via the DC/OS Web Interface
+
+1. Create a new service via the web interface in "Services" > "Deploy Service".
```

Comment:
Please make the UI text bold words, per our style guide: https://mesosphere.atlassian.net/wiki/display/DOCS/Mesosphere+Style+Guide.